src/js/schemas/service-schema/Networking.js

```diff
@@ -99,19 +99,19 @@ const Networking = {
             };
           }
 
-          let disabledLBPortFieldValue = 'Not Enabled';
+          let disabledLBPortFieldValue = service.lbPort || 'Not Enabled';
           if (prop === 'lbPort' && model && model.networking) {
             if (model.networking.networkType !== 'host') {
               definition.showLabel = 'Container Port';
-              if (definition.value === disabledLBPortFieldValue) {
+              if (definition.value === 'Not Enabled') {
```

Comment:
Same with this. Again, I just want to point it out. It has not been introduced in this PR, and does not need to be fixed here